package.json

```diff
@@ -2,34 +2,64 @@
 	"name": "lisk-js",
 	"version": "0.4.5",
 	"description": "JavaScript library for sending Lisk transactions from the client or server",
-	"main": "./dist-node/index.js",
+	"author": "Lisk Foundation <admin@lisk.io>, lightcurve GmbH <admin@lightcurve.io>",
+	"license": "GPL-3.0",
+	"keywords": [
+		"lisk",
+		"blockchain"
+	],
+	"homepage": "https://github.com/LiskHQ/lisk-js#readme",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/LiskHQ/lisk-js.git"
+	},
+	"bugs": {
+		"url": "https://github.com/LiskHQ/lisk-js/issues"
+	},
+	"engines": {
+		"node": "=6.12.3",
+		"npm": "=3.10.10"
+	},
+	"main": "dist-node/index.js",
 	"scripts": {
 		"prestart": "./bin/prestart.sh",
 		"start": "./bin/start.sh",
-		"test": "nyc mocha test --recursive",
-		"test:watch": "chokidar \"./(src|test)/**/*.js\" -c \"mocha test --recursive\" --initial",
+		"clean": "rm -rf ./node_modules ./dist-browser/lisk-js.js ./dist-browser/lisk-js.min.js ./dist-node",
+		"clean:browsertest": "rm -r browsertest/src browsertest/test",
+		"format": "prettier --write \"browsertest/{setup,runTests}.js\" \"{cypress/integration,src,test}/**/*.js\"",
+		"lint": "eslint .",
+		"lint:fix": "npm run lint -- --fix",
+		"test": "NODE_ENV=test nyc mocha test",
+		"test:watch": "npm test -- --watch",
```

Comment:
We were using chokidar because js-nacl broke mocha's watch functionality. This might have been fixed along the way with version upgrades, but we need to confirm. Does this still work ok?

Reply:
As i reverted this should be reverted too.
As long as `js-nacl` only applied once to the global, mocha watch works fine.
I will contact add that change here

Reply:
Ah great, thank you!